[Sprig App] Sprig CHIP-8

Milk-Cool opened the pull request:
### Author name
Author: Milk-Cool

### About your game

**What is your game about?**
A simple CHIP-8 emulator for Sprig.

Contains four ROMs:

https://johnearnest.github.io/chip8Archive/play.html?p=snek

https://johnearnest.github.io/chip8Archive/play.html?p=slipperyslope

https://johnearnest.github.io/chip8Archive/play.html?p=caveexplorer

https://johnearnest.github.io/chip8Archive/play.html?p=spockpaperscissors

**How do you play your game?**
WASD are mapped to 5789, K is mapped to 4 and L is mapped to 6.

You can swich between games using J and open the full sized keyboard with I (WASD to navigate, L to toggle)

You can upload your own ROMs by chaning the ROMS arrayi in the main file.

GalaxyGamingBoy commented:
Hi! Please fill out the `@author` and `@addedOn` tags with the relevant information so it can show up in the gallery with the new tag and the correct author. Also the `@description` tag is missing, add it to describe to others in the gallery what your game is about :D

whatwareweb commented:
ooh im gonna review this once i get through the backlog, this looks awesome

Milk-Cool commented:
hi again! i added the `KB_STICKY` option to fix the issue with Slippery Slope, which now holds the pressed keys until their status is queried by the CPU. (it's optional though, i've added the instructions on disabling it, too!) I also added the instructions in a comment at the top of the file, just like you asked.

thanks for the review! :3

whatwareweb reviewed (approved):
this is some really awesome work on this, thanks for making it :3

whatwareweb commented:
If you're in high school, you can get your sprig here! https://sprig-order.hackclub.dev/

also, people in the slack would love to see this! if you post a link and 1-2 sentences talking about your game to our [#ship](https://hackclub.slack.com/archives/C0M8PUPU6) [channel](https://hackclub.com/slack/), some people will probably play your game
